runtime/rasdump/dmpmap.c

```diff
@@ -121,7 +128,21 @@ static const DgDumpEnvSwitch dgSwitches[] = {
 			}
 };
 
+typedef struct OomDefaultSetting
+{
+	char *typeString;
+	char *onOpt;
```

Comment:
These fields should have type `const char *` because they refer to string constants.

Reply:
Done.